src/Backend/Modules/Blog/Domain/Comment.php

```diff
@@ -0,0 +1,205 @@
+<?php
+
+namespace Backend\Modules\Blog\Domain;
+
+use Common\Locale;
+use DateTime;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Table(name="blog_comments")
+ * @ORM\Entity(repositoryClass="Backend\Modules\Blog\Domain\CommentRepository")
```

Comment:
the file structure is wrong.
Should be in a directory Comment

Reply:
Fixed in 3cfb811e0b4fbd409e62096e036aa025efc55f88